### PR TITLE
Feature/add renderer preview by question path

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

File: src/features/courses/course-controller.ts [hunk 1]
```diff
@@ -1216,6 +1216,11 @@ class CourseController {
             calculatedRendererParameters.outputformat = OutputFormat.STATIC;
         }
 
+        // I do not like hardcoding the topicTypeId...
```
Comment (PanopticaRising) [inline src/features/courses/course-controller.ts:1219]: This might've been a merge that should've been deleted

Comment (PanopticaRising) [reply src/features/courses/course-controller.ts:1219]: @red-tt-bear @doombot-exe Can I get a confirmation before I remove it?

Comment (red-tt-bear) [reply src/features/courses/course-controller.ts:1219]: Not sure, this was @doombot-exe s code

Comment (red-tt-bear) [reply src/features/courses/course-controller.ts:1219]: Though if it's coming as an addition I would tend to agree

**Review by red-tt-bear — APPROVED** [review]

File: src/features/courses/course-controller.ts (hunk 1 above)
Comment (doombot-exe) [reply src/features/courses/course-controller.ts]: I moved it into another block

**Review by doombot-exe — CHANGES_REQUESTED** [review]
remove reassigning calculatedRendererParams (already happens at 1192)

**Review by doombot-exe — APPROVED** [review]
seems that's already been removed.

**Pull request merged**